bot.py

```diff
@@ -1,545 +1,860 @@
-        answer, reply_markup = apiquery(user_id, bssid, essid)
-    elif update.message.chat.type == 'private':
-        reply_keyboard = [['↪ Пропустить']]
-        update.message.reply_text(
-            'Укажите BSSID:',
-            reply_markup=ReplyKeyboardMarkup(reply_keyboard,
-                                             one_time_keyboard=True,
-                                             resize_keyboard=True)
-        )
-        context.user_data['sensivity'] = False
-        return ConversationStates.BSSID_PROMPT
-    update.message.reply_text(
-        answer, parse_mode='Markdown', reply_markup=reply_markup)
-    return ConversationHandler.END
-
-
-def bssid_prompt(update, context):
-    t = update.message.text
-    if re.match(bssid_pattern, t) is not None:
-        reply_keyboard = [['↪ Пропустить']]
-        context.user_data['bssid'] = t
-        update.message.reply_text(
-            'Укажите ESSID:',
-            reply_markup=ReplyKeyboardMarkup(reply_keyboard,
-                                             one_time_keyboard=True,
-                                             resize_keyboard=True)
-        )
-        return ConversationStates.ESSID_PROMPT
-    elif re.match(r'↪ ', t) is not None:
-        # Skip the BSSID prompt
-        context.user_data['bssid'] = '*'
-        update.message.reply_text(
-            'Укажите ESSID:', reply_markup=ReplyKeyboardRemove())
-        return ConversationStates.ESSID_PROMPT
+        r = requests.post(f"{SERVICE_URL}/api/apikeys", json = {
+            'login': users.data[str(message.from_user.id)]['key'].strip(),
+            'password': message.text,
+            'genread': True}).json()
+        if r['result']:
+            if r['profile']['level'] > 0:
+                nick = r['profile']['nick']
+                for k in r['data']:
+                    if k['access'] == 'read':
+                        key = k['key']
+                        break
+                users.mod(message, key = key)
+                await message.reply(lng.getmess(message, "logged_in"), parse_mode = "Markdown")
+                for user in users.security(users.data[str(message.from_user.id)]['key']):
+                    if user != str(message.from_user.id):
+                        await bot.send_message(user, lng.getmessbyuid(user, "security").format(
+                            name = f'{message.from_user.first_name if message.from_user.first_name else ""} {message.from_user.last_name if message.from_user.last_name else ""}',
+                            uid = f'`{message.from_user.id}`',
+                            username = f"@{message.from_user.username}" if message.from_user.username else ""), parse_mode = "Markdown")
+        elif r['error'] == 'loginfail':
+            users.mod(message, key = '-')
+            await message.reply(lng.getmess(message, "bad_cred"))
+        elif r['error'] == 'lowlevel':
+            users.mod(message, key = ' ')
```

Comment:
```suggestion
            users.mod(message, key = '-')
```
Исправлено, ломало авторизацию.